doc/api/fs.md

```diff
@@ -587,6 +587,8 @@ added: v0.1.10
 
 Returns `true` if the `fs.Stats` object describes a file system directory.
 
+This method is only valid when using [`fs.stat()`][] or [`fs.fstat()`][].
```

Comment:
```suggestion
If the `fs.Stats` object was obtains from [`fs.lstat()`][], this method will
always return false. This is because [`fs.lstat()`][] returns information
about a symbolic link itself and not the path it resolves to.
```